src/howtoplay/howtoplay.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Battleship - How to play?</title>
+  </head>
+  <body>
+    <main class="main">
+      <div class="container">
+        <h1 class="container__title">How to play?</h1>
+        <p class="container__instruction">
+          Battleship is a single-player game versus AI. Your goal is to be the
+          first to sink all of your opponent's ships. The game starts by
+          clicking the Start Game button and then the Randomize button, which
+          places your ships randomly on the board. The game is played on two
+          boards: yours and AI’s. Ships can be placed horizontally or vertically
+          and cannot overlap. They have different sizes: 1 of each 5-, 4-, 3-
+          square ship and 2 of each 2-, 1- square ships. The shot is made by
+          clicking on the desired square on the opponent's board. For each
+          successful hit on an AI’s ship, you get 1 point and an additional
+          point for sinking the ship. The best scores are recorded in high
+          scores. You can change the name displayed in the high scores by
+          entering your name and saving it in the Settings before the start of
+          the game, otherwise the score will be saved under the default name
+          Player.<br />Good luck, sailor!
```

Comment:
Don't use `<br />` in `<p>` tags.

https://developer.mozilla.org/pl/docs/Web/HTML/Element/p

Reply:
moved `<p>` tags do `<div>` tag with class container__instruction